stores/web3.ts

```diff
@@ -14,8 +11,8 @@ export const useWeb3Store = defineStore('web3', () => {
   // actions
   const addWeb3 = (
     providerName: string,
-    provider: Provider,
-    options?: HttpProviderOptions
+    provider: EIP1193Provider<Web3APISpec>,
+    _options?: any
```

Comment:
Maybe unknown instead of any? could we use a type that could be used for the `new Web3.providers.HttpProvider(provider)`

Reply:
That part needs bigger refactor, but I skipped that since we don't use it for any node. Apparently they've changed in new web3 http provider how headers are passed. It's no longer done through `options` param. I guess I will get back to this once it will be needed.